Sources/Overload/OvEditor/src/OvEditor/Panels/MenuBar.cpp

```diff
@@ -237,6 +237,8 @@ void OvEditor::Panels::MenuBar::CreateLayoutMenu()
 				{
 					auto& uiManager = *EDITOR_CONTEXT(uiManager);
 					EDITOR_EXEC(DelayAction(std::bind(&OvUI::Core::UIManager::SetLayout, &uiManager, entry.path()), 1));
+
+					Settings::EditorSettings::LatestLayout = entry.path().string();
```

Comment:
We could save just the filename (`.stem()`), so we can retrieve it afterwards at runtime, since there is no hierarchy inside of the `Layouts/` folder.

If the string `LatestLayout` is empty, then load default